src/users/cd-create-account.vue

```diff
@@ -208,42 +211,46 @@
   };
 </script>
 <style scoped lang="less">
-  @import '../common/styles/cd-primary-button'; 
+  @import "../common/styles/cd-primary-button";
+
   .cd-create-account {
     margin-right: 33px;
     margin-top: 50px;
     &__recaptcha{
       margin-top: 33px;
     }
     &__header {
-      background-color: #f4f5f6;
-      height: 100px;
-      text-align: center;
-      &-title {
-        margin-top: 21px;
-        font-size: 18px;
-        font-weight: bold;
-      }
-      &-info {
-        font-size: 16px;
-        margin-top: 4px;
+      font-size: 24px;
+      margin: 45px 0 16px 0;
+      font-weight: bold;
+      &--filler {
+        background-color: @cd-very-light-grey;
+        height: 24px;
+        width: 176px;
       }
     }
+    &__info {
+      margin-bottom: 16px;
+    }
     &__label {
-      margin-top: 16px;
-      display: block;
-      font-size: 16px;
-      font-weight: bold;
+      margin-bottom: 5px;
+      display: inline-block;
+      font-size: 14px;
+      font-weight: 700;
+    }
+    &__email {
+      max-width: 110%;
```

Comment:
Uh, what ?

Reply:
oh woop sorry i was playing around with that. will fix